patches/chrome-browser-net-stub_resolver_config_reader.cc.patch

```diff
@@ -1,12 +1,14 @@
 diff --git a/chrome/browser/net/stub_resolver_config_reader.cc b/chrome/browser/net/stub_resolver_config_reader.cc
-index 63035047782e814bb31acc18c610b61c91f740be..fdcc1c427a6709152b8ec7156e7b3b31364173d0 100644
+index 63035047782e814bb31acc18c610b61c91f740be..fcee3e09f4692ea368f187c507c50036878fb981 100644
 --- a/chrome/browser/net/stub_resolver_config_reader.cc
 +++ b/chrome/browser/net/stub_resolver_config_reader.cc
-@@ -370,6 +370,7 @@ SecureDnsConfig StubResolverConfigReader::GetAndUpdateConfiguration(
+@@ -369,7 +369,9 @@ SecureDnsConfig StubResolverConfigReader::GetAndUpdateConfiguration(
+   if (secure_dns_mode != net::SecureDnsMode::kOff) {
      doh_config = net::DnsOverHttpsConfig::FromStringLax(
          local_state_->GetString(prefs::kDnsOverHttpsTemplates));
++    BRAVE_GET_AND_UPDATE_CONFIGURATION
    }
-+  BRAVE_GET_AND_UPDATE_CONFIGURATION
++
```

Comment:
don't add empty lines in patches

Reply:
Fixed in https://github.com/brave/brave-core/commit/45d2b60ede86867c650abfeade1873d938d13f1f